### ISSUE TITLE
Missing information in audit_collector ?

### ISSUE THREAD

**Issue by LordFPL** [sscarduzio/elasticsearch-readonlyrest-plugin#328]
Hello,

I'm a bit surprised to see more information in logs than audit_collector : 

In logs : 
```
[2018-03-26T13:01:16,131][INFO ][t.b.r.a.ACL              ] FORBIDDEN by default req={ ID:838322889-401011158#293, TYP:SearchRequest, CGR:N/A, USR:[no basic auth header], BRS:true, ACT:indices:data/read/search, OA:ZZ.ZZ.ZZ.ZZ, DA:ZZ.ZZ.ZZ.ZZ, IDX:readonlyrest_audit-2018-03-26, MET:GET, PTH:/readonlyrest_audit-2018-03-26/_search, CNT:<N/A>, HDR:{Accept=*/*, Accept-Encoding=gzip, content-length=0, Forwarded=for=XX.XX.XX.XX; proto=https; by=YY.YY.YY.YY; httpproto=http/1.1, Host=es62.service.soa-dev.xxxx.fr, User-Agent=curl/7.29.0, X-Forwarded-For=XX.XX.XX.XX, XX.XX.XX.XX, X-Forwarded-Port=80, X-Forwarded-Proto=https, X-Real-Ip=XX.XX.XX.XX}, HIS:[Block 1 - J'aime le VLAN2 mais en GET seulement->[x_forwarded_for->false]], [::KIBANA-SRV::->[auth_key->false]], [::LDAP-RO::->[ldap_authentication->false]], [::APP-PROD::->[auth_key_sha256->false]] }
```

And in elasticsearch (via audit_collector) : 
```
{
  "_index": "readonlyrest_audit-2018-03-26",
  "_type": "ror_audit_evt",
  "_id": "838322889-401011158#293",
  "_version": 1,
  "found": true,
  "_source": {
    "error_message": null,
    "headers": [
      "Accept",
      "Accept-Encoding",
      "content-length",
      "Forwarded",
      "Host",
      "User-Agent",
      "X-Forwarded-For",
      "X-Forwarded-Port",
      "X-Forwarded-Proto",
      "X-Real-Ip"
    ],
    "acl_history": "[Block 1 - J'aime le VLAN2 mais en GET seulement->[x_forwarded_for->false]], [::KIBANA-SRV::->[auth_key->false]], [::LDAP-RO::->[ldap_authentication->false]], [::APP-PROD::->[auth_key_sha256->false]]",
    "origin": "ZZ.ZZ.ZZ.ZZ",
    "final_state": "FORBIDDEN",
    "task_id": 293,
    "type": "SearchRequest",
    "req_method": "GET",
    "path": "/readonlyrest_audit-2018-03-26/_search",
    "indices": [
      "readonlyrest_audit-2018-03-26"
    ],
    "@timestamp": "2018-03-26T13:01:16Z",
    "content_len_kb": 0,
    "error_type": null,
    "processingMillis": 3,
    "action": "indices:data/read/search",
    "id": "838322889-401011158#293",
    "content_len": 0,
    "user": null
  }
}
```

Information about "X-Forwarded-For=XX.XX.XX.XX" is missing (and others headers, but this one is important for me).

Is there anything to do to keep this information ?

**Comment by sscarduzio** [comment]
Just committed a fix that adds "xff" field (with x-forwarded-for header value) and a "destination" field (with the local ES IP).

**Comment by LordFPL** [comment]
Wow, that was fast ! :)
For information, i see the same thing for ldap user, present in log file, but not in elasticsearch. I can provide detail tomorrow.

Thx again ! :)

**Comment by LordFPL** [comment]
I see your commit, and you already correct it i think 👍
https://github.com/sscarduzio/elasticsearch-readonlyrest-plugin/commit/5aaffdf8a37b33b524fc2fbeb04c5e40711b5c47#diff-1b79c785b2a9a555eb8ff02837f5779fR329

Excellent... :-)

**Comment by LordFPL** [comment]
Fix ok in 1.16.19 (es6.4).
Thx again and all the best :)
